src/Sulu/Bundle/ContentBundle/Twig/ContentTwigExtension.php

```diff
@@ -138,7 +138,10 @@ public function needsAddButtonFunction(PropertyInterface $property)
      */
     public function isMultipleTest($property)
     {
-        return $property->getMinOccurs() > 1;
+        return
+            !is_null($property->getMaxOccurs()) ||
+            $property->getMinOccurs() > 1 ||
+            $property->getMinOccurs() < $property->getMaxOccurs();
```

Comment:
Exactly that's why I am worried so much about it 😅 Wouldn't it be easier if we take a different approach? E.g. just checking if maxOccurs is bigger than 1 (which would be clearly multiple) and handle the minOccurs=0 and maxOccurs=1 case in a different way?